src/RA_httpthread.cpp

```diff
@@ -252,31 +247,51 @@ void RAWeb::SetUserAgentString()
 #ifndef NTSTATUS
     #define NTSTATUS long
 #endif
-    using fnRtlGetVersion = NTSTATUS(NTAPI*)(PRTL_OSVERSIONINFOEXW lpVersionInformation);
-    auto RtlGetVersion = (fnRtlGetVersion)GetProcAddress(GetModuleHandleW(L"ntdll.dll"), "RtlGetVersion");
-    if (RtlGetVersion)
     {
-        RTL_OSVERSIONINFOEXW osVersion = {};
+        using fnRtlGetVersion = NTSTATUS(NTAPI*)(PRTL_OSVERSIONINFOEXW lpVersionInformation);
+
+        const auto ntModule{ ::GetModuleHandleW(L"ntdll.dll") };
+        if (!ntModule)
+            return;
```

Comment:
Need to not bail from this function if these fail. The User-Agent string should still be created, even if the exact NT version cannot be determined.

Reply:
OK.